README.md

```diff
@@ -662,7 +662,7 @@ API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [AXFR Database](http://api.axfrcheck.com) | AXFR public database | No | No | Unknown |
 | [FilterLists](https://filterlists.com/api) | Lists of filters for adblockers and firewalls | No | Yes | Unknown |
-| [HaveIBeenPwned](https://haveibeenpwned.com/API/v2) | Passwords which have previously been exposed in data breaches | No | Yes | Unknown |
+| [HaveIBeenPwned](https://haveibeenpwned.com/API/v3) | Passwords which have previously been exposed in data breaches | No | Yes | Unknown |
```

Comment:
Hello @VorBoto,

As mentioned in #1009, the API is not completely free anymore, so [it requires](https://haveibeenpwned.com/API/v3#Authorisation) providing an [API Key](https://haveibeenpwned.com/API/Key) for some endpoints.

```suggestion
| [HaveIBeenPwned](https://haveibeenpwned.com/API/v3) | Passwords which have previously been exposed in data breaches | `apiKey` | Yes | Unknown |
```

Reply:
Oh okay, I didn't dive in far enough to catch that.

Reply:
@VorBoto can you commit @Jamim suggestion? 🙂